packages/w/wasm-micro-runtime/xmake.lua

```diff
@@ -19,8 +20,8 @@ package("wasm-micro-runtime")
     add_configs("fast_interp", {description = "Enable fast interpreter", default = false, type = "boolean"})
     add_configs("aot", {description = "Enable AOT", default = false, type = "boolean"})
     -- TODO: improve llvm
-    add_configs("jit", {description = "Enable JIT", default = false, type = "boolean", readonly = true})
-    add_configs("fast_jit", {description = "Enable Fast JIT", default = false, type = "boolean", readonly = true})
+    add_configs("jit", {description = "Enable JIT", default = false, type = "boolean", readonly=true})
+    add_configs("fast_jit", {description = "Enable Fast JIT", default = false, type = "boolean", readonly=true})
```

Comment:
why? add spaces

Reply:
I tend to instinctively pad my operators, so I didn't really think much about it, sorry about that.